ui/src/timeMachine/utils/rawFluxDataTable.ts

```diff
@@ -19,7 +19,9 @@ export const parseFiles = (responses: string[]): ParseFilesResult => {
       data.push([])
     }
 
-    data.push(...parsedChunks[i])
+    for (let j = 0; j < parsedChunks[i].length; j++) {
```

Comment:
A comment would be useful here just to guard against any regressions in the future.